crates/rpc/rpc-testing-util/src/trace.rs

```diff
@@ -174,4 +174,51 @@ mod tests {
         let stream = client.trace_block_buffered(block, 2);
         assert_is_stream(&stream);
     }
+
+    async fn compare_endpoints(endpoint1: &str, endpoint2: &str, block_ids: Vec<BlockId>) {
```

Comment:
this is the right idea, but we want to have this as a standalone type.

this type should then zip the streams

https://docs.rs/futures/latest/futures/prelude/stream/trait.StreamExt.html#method.zip

and compare them one by one

Reply:
i think i got it.

Reply:
@mattsse i think i'm failling because i'm using the second rpc, i dunno which one i can use for this test